### PR TITLE
Add use case for Pay

### PR DESCRIPTION
**Pull request by TY1Fan**
Changes:

* Added use case for Pay in DG.
* Spotted and fixed a bug in previous use cases, i.e. use cases resumes should be use case resume instead.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -329,16 +329,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * 1a. System detects error in entered command.
     * 1a1. System displays error message and does not clear command line.
     * 1a2. User enters new command.
-
 * Steps 1a1-1a2 are repeated until all details entered are correct.
-* Use cases resumes from step 2.
+* Use case resumes from step 2.
```
Comment (Feng1231) [inline docs/DeveloperGuide.md:333]: Nice catch, thanks

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -394,13 +393,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+3. System calculates the tuition fee paid by the student for the lesson.
+4. System updates the total tuition fee paid by the student.
+5. System displays success message.
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. System cannot find the specified student.
+    * 1a1. User adds the student to the system (UC01).
```
Comment (Feng1231) [inline docs/DeveloperGuide.md:419]: should this be <ins> too?

**Review by Feng1231 — COMMENTED** [review]
Thanks, lgtm

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (TY1Fan) [reply docs/DeveloperGuide.md]: Thanks for catching this! Ive added in the <ins>

**Review by Feng1231 — APPROVED** [review]
lgtm

**Pull request merged**